vaadin-testbench-core/src/main/java/com/vaadin/testbench/unit/BaseUIUnitTest.java

```diff
@@ -215,4 +215,16 @@ private <T extends ComponentWrap<Y>, Y extends Component> T initialize(
                             .getSimpleName());
         }
     }
+
+    /**
+     * Simulates a server round-trip, flushing pending component changes.
+     */
+    protected static void roundTrip() {
```

Comment:
Let's keep this protected for now and try to automate where ever it is needed, but it might be that we need this as public api.